cafebabel/core/mixins.py

```diff
@@ -0,0 +1,48 @@
+from flask import current_app
+from mongoengine import signals
+
+from .. import db
+
+
+class UploadableImageMixin:
+    has_image = db.BooleanField(default=False)
```

Comment:
Not fond of having fields declared in many places, I would keep that one on `Article` and thus get rid of the `db` dependency in the mixins.

Reply:
It' about to be duplicated for User that will have the same field. that's why the split into a mixin.
Plus the Mixin is explicitely dependant of that field.

Reply:
Fair enough, I wasn't aware of the future use in User 👍